index.d.ts

```diff
@@ -1,93 +1,91 @@
-   */
-  export interface NodeTypeAnnotations {
-    /** The type annotation of the Node */
-    name: string;
-    /** The type annotations of the {@link https://github.com/kdl-org/kdl/blob/main/SPEC.md#argument|Arguments} */
-    values: string[];
-    /** The type annotations of the {@link https://github.com/kdl-org/kdl/blob/main/SPEC.md#property|Properties} */
-    properties: Record<string, string>;
-  }
+/**
+ * The {@link https://github.com/kdl-org/kdl/blob/main/SPEC.md#type-annotation|Type annotations} associated with a {@link https://github.com/kdl-org/kdl/blob/main/SPEC.md#node|Node}.
+ */
+export interface NodeTypeAnnotations {
+  /** The type annotation of the Node */
+  name: string;
+  /** The type annotations of the {@link https://github.com/kdl-org/kdl/blob/main/SPEC.md#argument|Arguments} */
+  values: string[];
+  /** The type annotations of the {@link https://github.com/kdl-org/kdl/blob/main/SPEC.md#property|Properties} */
+  properties: Record<string, string>;
+}
 
-  /**
-   * A {@link https://github.com/kdl-org/kdl/blob/main/QUERY-SPEC.md|Query string}.
-   */
-  export type QueryString = string;
+/**
+ * A {@link https://github.com/kdl-org/kdl/blob/main/QUERY-SPEC.md|Query string}.
+ */
+export type QueryString = string;
 
-  export interface ParseResult {
-    /** Parsing errors */
-    errors: chevrotain.IRecognitionException[];
-    /** KDL Document */
-    output?: Document;
-  }
+export interface ParseResult {
+  /** Parsing errors */
+  errors: chevrotain.IRecognitionException[];
+  /** KDL Document */
+  output?: Document;
+}
 
-  /**
-   * Formatting options
-   */
-  export interface FormattingOptions {
-    escapes?: Record<number, boolean>;
-    requireSemicolons?: boolean;
-    escapeNonAscii?: boolean;
-    escapeNonPrintableAscii?: boolean;
-    escapeCommon?: boolean;
-    escapeLinespace?: boolean;
-    newline?: string;
-    indent?: number;
-    indentChar?: string;
-    exponentChar?: string;
-    printEmptyChildren?: boolean;
-    printNullArgs?: boolean;
-    printNullProps?: boolean;
-  }
+/**
+ * Formatting options
+ */
+export interface FormattingOptions {
+  escapes?: Record<number, boolean>;
+  requireSemicolons?: boolean;
+  escapeNonAscii?: boolean;
+  escapeNonPrintableAscii?: boolean;
+  escapeCommon?: boolean;
+  escapeLinespace?: boolean;
+  newline?: string;
+  indent?: number;
+  indentChar?: string;
+  exponentChar?: string;
+  printEmptyChildren?: boolean;
+  printNullArgs?: boolean;
+  printNullProps?: boolean;
 }
 
 /**
  * @param {string} text - Input KDL file (or fragment)
  */
-export function parse(text: string): kdljs.ParseResult;
+export function parse(text: string): ParseResult;
 
 /**
- * @param {kdljs.Document} doc - Input KDL document
+ * @param {Document} doc - Input KDL document
  */
-export function format(doc: kdljs.Document, options?: kdljs.FormattingOptions): string;
+export function format( doc: Document, options?: FormattingOptions,): string;
```

Comment:
Why the space and trailing comma here?

Reply:
Not sure. I re-ran prettier and it cleaned them up though.